libs/platform/src/lib/components/form/multi-input/base-multi-input.ts

```diff
@@ -136,10 +127,6 @@ export abstract class BaseMultiInput extends CollectionBaseInput implements Afte
     @Input()
     secondaryTextAlignment: TextAlignment = 'right';
 
-    /** Turns on/off Adjustable Width feature */
```

Comment:
was this an intentional input remove? If so then it has to be a breaking change PR